docs/source/index.rst

```diff
@@ -0,0 +1,13 @@
+Welcome to git-machete's documentation!
+=======================================
+Probably the sharpest git repository:
```

Comment:
Pls extend the slogan to the full form... it doesn't make sense in the current shape ;)